runners/core-java/src/main/java/org/apache/beam/runners/core/PaneInfoTracker.java

```diff
@@ -55,6 +55,12 @@ public void clear(StateAccessor<?> state) {
     state.access(PANE_INFO_TAG).clear();
   }
 
+  @SuppressFBWarnings(value = "RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT",
```

Comment:
Checkstyle and builds work without this so I removed it.  I had copied it from somewhere else but perhaps it's no longer necessary.